crates/bevy_render/src/lib.rs

```diff
@@ -58,19 +58,22 @@ pub mod prelude {
     #[doc(hidden)]
     pub use crate::{
         alpha::AlphaMode,
-        camera::{
-            Camera, ClearColor, ClearColorConfig, OrthographicProjection, PerspectiveProjection,
-            Projection,
-        },
+        camera::ToNormalizedRenderTarget as _,
```

Comment:
Re-export to minimize diff. Remove in a follow-up if we don't want it here, i think its probably fine though.